src/clr-core/styles/tokens/tokens.yml

```diff
@@ -44,17 +52,29 @@ props:
     type: 'px'
     category: 'space'
   - name: 'size-8'
-    value: '24px'
+    value: '20px'
```

Comment:
As per slack do you want to bump this down to 18px then that will meet the quarter values of 24px? I think thats all we would need left for the values